bin/wiki_entity_linking/wikipedia_processor.py

```diff
@@ -493,15 +494,12 @@ def read_el_docs_golds(nlp, entity_file_path, dev, line_ids, kb, labels_discard=
                 if dev != is_dev(article_id) or not is_valid_article(clean_text):
                     continue
 
-                texts.append(clean_text)
-                entities_list.append(entities)
-
-    docs = nlp.pipe(texts, batch_size=50)
-
-    for doc, entities in zip(docs, entities_list):
-        gold = _get_gold_parse(doc, entities, dev=dev, kb=kb, labels_discard=labels_discard)
-        if gold and len(gold.links) > 0:
-            yield doc, gold
+                doc = nlp(clean_text)
```

Comment:
Not sure if it matters for efficiency here, but it probably still makes sense to use `nlp.pipe`? And maybe wrap the line loading logic in its  own iterator that `nlp.pipe` an consume?

Reply:
@svlandeg Do we need to change this? If not I'd go ahead and merge. 👍

Reply:
Let's hold of on the merge for a second, I'll implement Ines' suggestion, it probably will be more efficient

Reply:
I've implemented this idea locally, but it doesn't really seem to speed things up further. What's more, is because the `nlp.pipe()` needs to wait for a batch, the progress bar seems be stuck at 0% again for some time in the beginning. So if OK by you, I would just keep this PR as it is now.